components/BottomNavigation/examples/BottomNavigationBarControllerExampleViewController.swift

```diff
@@ -18,6 +18,44 @@ import MaterialComponentsBeta.MaterialBottomNavigationBeta
 import MaterialComponents.MaterialBottomNavigation_ColorThemer
 import MaterialComponents.MaterialBottomNavigation_TypographyThemer
 
+class BottomNavigationControllerExampleScrollableChildViewController : UICollectionViewController {
+
+  let numberOfItems = 200
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    collectionView.dataSource = self
+    collectionView.register(UICollectionViewCell.self, forCellWithReuseIdentifier: "cell")
+
+    if #available(iOS 11.0, *) {
+      collectionView.contentInsetAdjustmentBehavior = .always
+      collectionView.insetsLayoutMarginsFromSafeArea = true
+    }
+  }
+
+  override func numberOfSections(in collectionView: UICollectionView) -> Int {
+    return 1
+  }
+
+  override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    return numberOfItems
+  }
+
+  override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+    let cellColors: NSArray = {
```

Comment:
You probably want to move this array creation out of `cellForItemAt` – the entire array is being recomputed for every cell. 

Alternatively, it might be more straightforward to define the cell's bg color's hue as 1/indexPath.item

Reply:
Done.